Otus-DevOps-2020-02/DmitryKorlas_microservices
# Kubernetes 5

# Completed the homework №29


## What were done:
 - add Prometheus chart
 - add custom_values in prometheus chart
 - enable kubeStateMetrics and nodeExporter in prometheus
 - add job reddit-endpoint
 - add kubernetes labels for reddit-endpoints job
 - add job reddit-production
 - split reddit-endpoints into 3 separate jobs - comment-endpoints, ui-endpoints, post-endpoints
 - update README.md - grafana instructions

## How to run the project:
 - setup Grafana+Prometheus, run reddit app using helm. see README.md for details

## How to check, that it works correctly:
 - observe http://reddit-prometheus/targets - it should contains ui-endpoints, post-endpoints and comment-endpoints targets
 - observe grafana dashboards at http://reddit-grafana/

## PR checklist:
 - [x] Added a label with the number of homework
